src/Proxy/LoggerProxy.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Proxy;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Yiisoft\Yii\Debug\Collector\CollectorInterface;
+
+final class LoggerProxy implements LoggerInterface
+{
+    private LoggerInterface $logger;
+    private CollectorInterface $collector;
+
+    public function __construct(LoggerInterface $logger, CollectorInterface $collector)
```

Comment:
I think it is better to use LogCollector here since no other collector can handle it.

Reply:
`LogCollector` is marked as `final` so `LoggerProxy` can't be configured by another `Collector`, but should, I think

Reply:
OK. Let it be for now...